index.js

```diff
@@ -162,6 +164,58 @@ const inventoryCommand = (message) => {
   });
 };
 
+const lookUpCommand = (message) => {
+  const originalMessage = message.content;
+  const fuzzySearchKey = originalMessage.split(" ").splice(1).join(" ");
+  const fuzzySearchUrl =
+    "https://db.ygoprodeck.com/api/v7/cardinfo.php?&fname=";
+
+  console.log(fuzzySearchUrl + fuzzySearchKey);
+
+  axios
+    .get(fuzzySearchUrl + fuzzySearchKey)
+    .then((cardSearchResults) => {
+      const objectsReturnedBySearch = cardSearchResults.data.data;
```

Comment:
this actually a good name, but better if you can name the object. In this case, we know the object is going to be a card, so could be something like `searchResultCards`. The last part of a name signifies what kind of object you're dealing with.

Also naming sounds like one of those unimportant things, but at this point its become a meme as [the 2nd hardest problem in programming](https://martinfowler.com/bliki/TwoHardThings.html). You'd be surprised how much time in pull requests is spent going back and forth on names 😏